src/builder.js

```diff
@@ -269,7 +260,9 @@ export default function builder() {
     }
 
     function buildPayload(err) {
-      let data = cgiData();
+      const data = {};
+      if (err.userAgent) { data['HTTP_USER_AGENT'] = err.userAgent; }
+      if (err.referrer) { data['HTTP_REFERER'] = document.referrer; }
```

Comment:
Should this assignment be from `err.referrer` not `document.referrer`?

Reply:
Ah, good catch. Yes it should. :)

Reply:
@rabidpraxis fixed, thanks!